src/elements/content-sidebar/activity-feed/comment/Comment.js

```diff
@@ -249,6 +255,7 @@ class Comment extends React.Component<Props, State> {
                             />
                         ) : (
                             <ActivityMessage
+                                canCollapse={allowCollapse}
```

Comment:
Should this be handled within `ActivityMessage` so annotation messages also get the same experience?

Reply:
I mean, I don't know. It can. Probably would be better for the users.